BetaCameras/RealSense2/RealSense2.cs

```diff
@@ -1366,51 +1357,37 @@ protected override void ConnectImpl()
             }
 
             StartPipeline();
+            Device dev = GetDevice();
+            _model = dev.Info[CameraInfo.Name];
 
-            if(!haveSerial)
-            {
-                this.SerialNumber = RealSense2API.GetDeviceInfo(_pipeline, RealSense2API.CameraInfo.SERIAL_NUMBER);
-            }
+            if (!haveSerial)
+                this.SerialNumber = dev.Info[CameraInfo.SerialNumber];
```

Comment:
What's the difference between this line and the `Firmware` property?
I suspect an error in the Firmware property. If it really is the same, why keep Firmware?

Reply:
The `Firmware` property should return `Info[CameraInfo.FirmwareVersion]`. Good catch. Too much mindless updating of code leads to a lot of slip ups =/